sdk/core/azure-core/test/ut/operation_test.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/context.hpp>
+#include <azure/core/nullable.hpp>
+#include <azure/core/operation.hpp>
+#include <azure/core/operation_status.hpp>
+#include <azure/core/response.hpp>
+
+using namespace Azure::Core;
+
+namespace Azure { namespace Core { namespace Test {
+
+  class StringClient;
+
+  class StringOperation : public Operation<std::string> {
+
+  private:
+    StringClient* m_client;
+    std::string m_operationToken;
+    std::string m_value;
+
+  private:
+    int count = 0;
+
+  public:
+    StringOperation(StringClient* client){ m_client = client; }
+
+
+    std::string GetResumeToken() { return m_operationToken; }
+
+    std::unique_ptr<Http::RawResponse> Poll(Context& context = Context())
```

Comment:
Ditto "`Context& context = Context()` is no bueno". Also, it's good practice to annotate these virtual functions with `override` to indicate that they are overriding some virtual signature from a base class: it tells readers there's action-at-a-distance going on, and tells compilers to error if we failed to properly override the base's signature.
```suggestion
    std::unique_ptr<Http::RawResponse> Poll(Context& context) override
```